contribute/hacker-guide.md

```diff
@@ -349,7 +302,7 @@ This means your change is backward or forward binary incompatible with the speci
 
 Now to make sure that my fix doesn't break anything I need to run the test suite using the `partest` tool we wrote to test Scala.
 Read up [the partest guide](partest-guide.html) to learn the details about partest, but in a nutshell you can either
-run `ant test` to go through the entire test suite (30+ minutes) or use wildcards to limit the tests to something manageable:
+run `sbt test` to go through the entire test suite (30+ minutes) or use wildcards to limit the tests to something manageable:
```

Comment:
`sbt test` only runs the JUnit tests, to run the entire test suite, one also needs to do `sbt partest` (for now, this might get better..) see https://github.com/scala/scala#using-the-sbt-build